drivers/bridge/bridge.go

```diff
@@ -1009,6 +1012,10 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	if err := d.storeUpdate(endpoint); err != nil {
+		logrus.Warnf("Failed to save bridge endpoint %s to store: %v", ep.id[0:7], err)
```

Comment:
We can fail, just that we will fail even if the user did not start the daemon with the restore flag.
Anyhow, I think that if the save to store fails for this, it would fail for many others, so I do not think the system would be usable anyway. I will make the change.